### PR TITLE
Revamp the connector

### PR DESCRIPTION
**Pull request by kalaiyarasiganeshalingam**
## Purpose
This PR has revamped this connector with AWS SDK 2.x

### CODE REVIEW

File: src/main/java/org/wso2/carbon/connector/amazonsqs/operations/message/ChangeMessageVisibility.java [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+                    Constants.VISIBILITY_TIME_OUT);
+            String apiCallTimeout = (String) ConnectorUtils.lookupTemplateParamater(messageContext,
+                    Constants.API_CALL_TIMEOUT);
+            String apiCallAttemptTimeout = (String) ConnectorUtils.lookupTemplateParamater(messageContext,
+                    Constants.API_CALL_ATTEMPT_TIMEOUT);
+            ChangeMessageVisibilityRequest.Builder changeMessageVisibilityRequestBuilder =
+                    ChangeMessageVisibilityRequest.builder().
+                    queueUrl(queueUrl).visibilityTimeout(Integer.valueOf(visibilityTimeout)).
+                    receiptHandle(receiptHandle);
+            if (StringUtils.isNotEmpty(apiCallTimeout) || StringUtils.isNotEmpty(apiCallAttemptTimeout)) {
```
Comment (SanojPunchihewa) [inline src/main/java/org/wso2/carbon/connector/amazonsqs/operations/message/ChangeMessageVisibility.java:64]: Shall we use `StringUtils.isNotBlank` to handle whitespaces as well? Please change the other occurrences as well.

File: src/main/java/org/wso2/carbon/connector/amazonsqs/connection/ConnectionConfiguration.java [hunk 2]
```diff
@@ -63,20 +71,90 @@ public void setAwsAccessKeyId(String awsAccessKeyId) {
     }
 
     public String getAwsSecretAccessKey() {
-        return awsSecretAccessKey;
+        return this. awsSecretAccessKey;
```
Comment (SanojPunchihewa) [inline src/main/java/org/wso2/carbon/connector/amazonsqs/connection/ConnectionConfiguration.java:74]:
```suggestion
        return this.awsSecretAccessKey;
```

File: src/main/java/org/wso2/carbon/connector/amazonsqs/operations/queue/ListQueues.java [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+        } catch (NumberFormatException e) {
+            Utils.setErrorPropertiesToMessage(messageContext, Error.INVALID_CONFIGURATION,
+                    Constants.NUMBER_FORMAT_ERROR_MSG + e.getMessage());
+            handleException(Constants.NUMBER_FORMAT_ERROR_MSG, e, messageContext);
+        } catch (Exception e) {
+            Utils.setErrorPropertiesToMessage(messageContext, Error.GENERAL_ERROR, e.getMessage());
+            handleException(Constants.GENERAL_ERROR_MSG + e.getMessage(), messageContext);
+        }
+    }
+}
```
Comment (arunans23) [inline src/main/java/org/wso2/carbon/connector/amazonsqs/operations/queue/ListQueues.java:98]: Shall we add a new line here?

**Review by arunans23 — APPROVED** [review]
LGTM

**Review by arunans23 — APPROVED** [review]

**Pull request merged**